src/OpenLoco/src/Ui.cpp

```diff
@@ -333,18 +333,6 @@ namespace OpenLoco::Ui
         SDL_ShowCursor(1);
     }
 
-    // 0x0040447F
-    void initialiseInput()
-    {
-        call(0x0040447F);
```

Comment:
Are there associated hooks or stubs that can be removed as well

Reply:
No direct address matches. We've probably stubbed quite a few of the Win32 calls involved, but I think we decided to drop them as once when the time comes, right?